tools/AutomationTestFramework/RunBooks/LiveStorageTests.ps1

```diff
@@ -0,0 +1,30 @@
+Param(
```

Comment:
Should these be checked in?  These look like the generated runbooks created by the scripts?

Reply:
removed.